store/src/java/com/zimbra/cs/volume/VolumeCLI.java

```diff
@@ -256,27 +257,35 @@ private void print(VolumeInfo vol) {
         System.out.println("                        type: " + toTypeName(vol.getType()));
         System.out.println("                        path: " + vol.getRootPath());
         System.out.print("                  compressed: " + vol.isCompressBlobs());
-        
+
         if (vol.isCompressBlobs()) {
             System.out.println("\t                 threshold: " + vol.getCompressionThreshold() + " bytes");
         } else {
             System.out.println();
         }
         System.out.println("                     current: " + vol.isCurrent());
-        if(vol.getStoreType() == 2) {
-            VolumeExternalInfo volumeExternalInfo = vol.getVolumeExternalInfo();
-            if (A_VOLUME_S3.equals(volumeExternalInfo.getStorageType())) {
+        if (vol.getStoreType() == 2) {
```

Comment:
We have constant/enum for `2` right?

Reply:
We don't have any constant enum for 2 or 1, we were going to write in new repo but we didnt had that conts

Reply:
see this https://github.com/Zimbra/zm-mailbox/blob/feature/9.1.0/store/src/java/com/zimbra/cs/volume/Volume.java#L71

Reply:
Ok. fixed